.github/workflows/tests.yml

```diff
@@ -10,6 +10,9 @@ jobs:
       matrix:
         os: [ubuntu-16.04, ubuntu-18.04, ubuntu-20.04]
         python: [3.5, 3.6, 3.7, 3.8]
```

Comment:
I actually think 3.6 isn't available on `Focal` either.

Reply:
Doesn't seem like any Python version will install on the 20.04 image yet. It's a preview after all, so I'll disable it for now.

Reply:
autopkgtest was an unknown command on 16.04 for some reason, passed on 18.04 and 20.04.